llvm/tools/sycl-post-link/SpecConstants.cpp

```diff
@@ -620,15 +716,24 @@ PreservedAnalyses SpecConstantsPass::run(Module &M,
   for (const auto &P : SCMetadata)
     MD->addOperand(P.second);
 
+  NamedMDNode *MDDefaults =
+      M.getOrInsertNamedMetadata(SPEC_CONST_DEFAULT_VAL_MD_STRING);
+  for (const auto &P : DefaultsMetadata)
+    MDDefaults->addOperand(P);
+
   return IRModified ? PreservedAnalyses::none() : PreservedAnalyses::all();
 }
 
-bool SpecConstantsPass::collectSpecConstantMetadata(Module &M,
-                                                    SpecIDMapTy &IDMap) {
+bool SpecConstantsPass::collectSpecConstantMetadata(
+    Module &M, SpecIDMapTy &IDMap, std::vector<char> &DefaultValues) {
   NamedMDNode *MD = M.getOrInsertNamedMetadata(SPEC_CONST_MD_STRING);
   if (!MD)
     return false;
 
+  NamedMDNode *N = M.getOrInsertNamedMetadata(SPEC_CONST_DEFAULT_VAL_MD_STRING);
```

Comment:
Wow, we should use just `get`, no need to insert anything here, but I guess it was introduced by me on line 729